Release v7.1.0 (#142)
newrelic/node-test-utilities: CHANGELOG.md

### v7.1.0 (2022-09-14)

> * Added the capacity of running ESM module tests with the ESM loader in `node-newrelic` or a path to an ESM loader set as process.env.NR_LOADER.

Reviewer: ```suggestion
* Added the capacity of running ES module tests with the ESM loader from agent or a path to an ESM loader set as process.env.NR_LOADER.
```
